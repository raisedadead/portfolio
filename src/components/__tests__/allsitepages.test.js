import React from 'react'
import renderer from 'react-test-renderer'
import AllSitePages from '../allsitepages'

describe(`AllSitePages`, () =>
  it(`renders correctly -- according to react-test-renderer`, () => {
    const tree = renderer.create(<AllSitePages />).toJSON()
    expect(tree).toMatchSnapshot()
  })
)
