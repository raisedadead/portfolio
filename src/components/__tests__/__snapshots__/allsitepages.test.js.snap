// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllSitePages renders correctly -- according to react-test-renderer 1`] = `
<section>
  <h2>
    About thisÂ Starter
  </h2>
  <nav>
    <ul>
      <li
        id="home"
      >
        <a
          activeClassName="current"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        id="dashboard"
      >
        <a
          activeClassName="current"
          href="/dashboard/"
        >
          Client-Only Path
        </a>
      </li>
      <li
        id="fourOhFour"
      >
        <a
          activeClassName="current"
          href="/404/"
        >
          Custom 404 Page
        </a>
      </li>
      <li
        id="plugins"
      >
        <a
          activeClassName="current"
          href="/plugins/"
        >
          Plugins
        </a>
      </li>
      <li
        id="tools"
      >
        <a
          activeClassName="current"
          href="/tools/"
        >
          Tools and Utilities
        </a>
      </li>
    </ul>
  </nav>
</section>
`;
